src/main/java/seedu/canoe/logic/commands/FindAllTrainingCommand.java

```diff
@@ -0,0 +1,66 @@
+package seedu.canoe.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.canoe.commons.core.Messages;
+import seedu.canoe.logic.commands.exceptions.CommandException;
+import seedu.canoe.model.Model;
+import seedu.canoe.model.student.IdMatchesPredicate;
+import seedu.canoe.model.student.TrainingMatchesIdPredicate;
+
+/**
+ * Finds and lists all trainings in canoe book which match the student id specified.
+ */
+public class FindAllTrainingCommand extends Command {
```

Comment:
Should it be called `FindStudentTrainingCommand`? Idk. This name sounds like you want all trainings instead of a particular student's training schedules.